docs/guides/connecting_to_your_data/overview.md

```diff
@@ -0,0 +1,205 @@
+
+If you use the Great Expectations CLI, you can run this command to automatically generate a pre-configured Jupyter Notebook:
+
+```console
+great_expectations datasource new
+```
+
+From there, you will be able to follow along a YAML based workflow for configuring and saving your Datasource.  Whether you prefer to work with the Jupyter Notebook's boilerplate for creating a datasource, or would rather dive in from scratch with a Python script, however, most of the work will take place in the configuring of the Datasource in question.
+
+
+### 2. Configure your Datasource
+
+Because the underlying data systems are different, configuration for each type of Datasource is slightly different.  We have step by step how-to guides that cover many common cases, and core concepts documentation to help you with more exotic kinds of configuration.  It is strongly advised that you find the guide that pertains to your use case and follow it.  If you are simply interested in learning about the process, however, the following will give you a broad overview of what you will be doing regardless of what your underlying data systems are.
+
+Datasource configurations can be written as YAML files or Python dictionaries.  Regardless of variations due to the underlying data systems, your Datasource's configuration will look roughly like this:
+
+<Tabs
+  groupId="yaml-or-python"
+  defaultValue='yaml'
+  values={[
+  {label: 'YAML', value:'yaml'},
+  {label: 'Python', value:'python'},
+  ]}>
+  <TabItem value="yaml">
+
+```python
+datasource_yaml = fr"""
+name: <name_of_your_datasource>
+class_name: Datasource
+execution_engine:
+    class_name: <class_of_execution_engine>
+data_connectors:
+    <name_of_your_data_connector>:
+        class_name: <class_of_data_connector>
+        <additional_keys_based_on_source_data_system>: <corresponding_values>
+"""
+```
+
+</TabItem>
+<TabItem value="python">
+
+```python
+datasource_config = {
+    "name": "<name_of_your_datasource>",
+    "class_name": "Datasource",
+    "execution_engine": {"class_name": "<class_of_execution_engine>"},
+    "data_connectors": {
+        "<name_of_your_data_connector>": {
+            "class_name": "<class_of_data_connector>",
+            "<additional_keys_based_on_source_data_system>": "<corresponding_values>"
+        }
+    }
+}
+
+```
+
+</TabItem>
+</Tabs>
+
+Please note that this is just a broad outline of the configuration you will be making.  You will find much more detailed examples in our documentation on how to connect to specific source data systems.
+
+The `name` and `class_name` top level keys will be the first you need to define.  The `name` key can be anything you want, but it is best to use a descriptive name as you will use this to reference your Datasource in the future. Unless you are extending Great Expectations and using a subclass of Datasource, you will almost never need to use a `class_name` other than `Datasource` for the top level `class_name` value.
```

Comment:
I don't know what our current state is with `SimpleSqlAlchemyDatasource`, so you may need to confirm this with @alexsherstinsky or @donaldheppner.

Reply:
Yeah, I suspect that `Datasource` will rarely be the most convenient way class to use. Users will usually do better with `PandasDatasource`, `SparkDFDatasource`, `SqlAlchemyDatasource`, `SimpleSqlAlchemyDatasource`, etc.

Reply:
Hm. After reviewing, `PandasDatasource` and `SparkDFDatasource` are v2, not v3. I suspect that it would be relatively simple to create comparable classes in V3---which would save the user a lot of boilerplate configuration---but I haven't looked closely.

Reply:
@Rachel-Reverie @talagluck @abegong @donaldheppner -- I agree with Abe in terms of what is v2 and what is v3.  In addition, I would like to reiterate that we need to allocate some development cycles to certify the use of `InferredAssetSqlDataConnector` and `ConfiguredAssetSqlDataConnector` (and make any updates to accomplish this certification) and the `Datasource` using these direct SQL data connectors.

As part of this effort, we will also document the recommended usage for `SimpleSqlalchemyDatasource`.  Today, all documented cases of the direct SQL data connectors are not certified, not used in CLI, while only `SimpleSqlalchemyDatasource` is officially certified (but appears in documentation a lot more than it should).

We also need to improve the way `Datasource` is created for and displayed in Jupyter notebooks generated by `CLI datasource` and `CLI suite` commands -- today, the nuance between `BatchRequest` and `RuntimeBatchRequest` is not handled at all (in addition to the `Datasource` not having the optimal data connector for the given CLI operation.

I was pretty sure that I created JIRA items for these issues before, but now, unfortunately, I am unable to find them.  Hence, before we re-enter them, perhaps someone with stronger Atlassian privileges could search for them.

Happy to discuss.  Thank you.